mod-source-record-manager-server/src/main/java/org/folio/services/JobExecutionServiceImpl.java

```diff
@@ -509,7 +509,7 @@ private JobExecution modifyJobExecutionToCompleteWithError(JobExecution jobExecu
         } else {
           String message = format("Records from SRS were not deleted for JobExecution %s", jobExecutionId);
           LOGGER.error(message);
-          promise.fail(new HttpStatusException(response.result().statusCode(), message));
+          promise.fail(new HttpException(response.result().statusCode(), "Error during post for new Snapshot."));
```

Comment:
@bisecomsergiy Why error message was changed?

Reply:
Very good point, moved back it. Thank you!